### DISCUSSION TITLE
Your GitHub Copilot access has been disabled

### DISCUSSION THREAD

**Discussion by sdudnic-ingerop** [community/community#118516] [Copilot Conversations]
### Select Topic Area

Question

### Body

having Copilot Business, we have an organization with three admins, no one deactivated the copilot, but suddenly we got an email "Your GitHub Copilot access has been disabled", and "ask one of the owners of your organizations"... however I am one of owners, when I try to validate the payment on this page
https://github.com/github-copilot/business_signup/organization/payment?org=mycompany

I get 404 error... 

![image](https://github.com/community/community/assets/148953528/48679b34-6503-4355-8fbf-10abff047229)

TODO: 
- Fix the 404 bug
- If there is a problem with, say, the card inform users that there is a problem with payment, and if not solved will be disconnected, not just disconnect users like this without any warning or exaplanation

**Comment by sdudnic-ingerop** [comment]
the problem was with the card, however that does not explain multiple bugs in the githib interface in that case.

**Comment by dineshke-syn** [comment]
Has anyone received a response from GitHub Support about this?

Were you able to get credits/refunds for the duplicate charges?

Did you find any workaround to prevent the automatic deallocation?
